recipe-client-addon/lib/Utils.jsm

```diff
@@ -0,0 +1,14 @@
+this.EXPORTED_SYMBOLS = ["jsonCopy"];
+
+/**
+ * Copy a simple object by converting it to JSON and back.
+ * @param  {Object} value Value to clone
+ * @return {Object}       A deep copy of the value, or the original
+ *                        value if it's not an object.
+ */
+this.jsonCopy = function(value) {
+  if (value !== null && typeof value === "object") {
+    value = JSON.parse(JSON.stringify(value));
```

Comment:
Nit: can just use: `Cu.cloneInto(value, {});` instead of going through JSON. :-)